src/rabbitmq.js

```diff
@@ -179,16 +236,21 @@ class RabbitMQ {
 
   /**
    * Takes an object representing a message and sends it to a task queue.
-   *
+   * appends passed in name to tasks
    * @param {String} queue Task queue to receive the message.
    * @param {Object} content Job to send.
    * @return {Promise} Promise resolved when message is sent to queue.
    */
   publishTask (queue: string, content: Object): Bluebird$Promise<void> {
     return Promise.try(() => {
+      const queueName = `${this.name}.${queue}`
```

Comment:
yes, we are going to start prefixing tasks with `APP_NAME` this is to namespace tasks based on applications